src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -1,4 +1,27 @@
 package com.kodilla.ecommercee.controller;
 
+import com.kodilla.ecommercee.entity.GenericEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value = "/users", produces = "application/json")
 public class UserController {
+    @PostMapping("/createUser")
+    public void createUser(@RequestBody GenericEntity userDto){
+        new GenericEntity();
+    }
+
+    @GetMapping("/{id}")
+    public GenericEntity getUser(@RequestParam Long userId){
```

Comment:
dlaczego `@RequestParam` a nie `@PathVariable`

Reply:
Sugerowałam się tym, co jest w zakresie kursu (tam jest używane @RequestParam). Znalazłam też coś takiego:
"The key difference between @RequestParam and @PathVariable is that @RequestParam used for accessing the values of the query parameters where as @PathVariable used for accessing the values from the URI template."
I zasugerowałam się, że wobec tego w tym przypadku powinnam użyć @RequestParam. Czy to jest błąd?